### DISCUSSION TITLE
Certification Criteria

### DISCUSSION THREAD

**Discussion by AndrewRCD** [rrirower/DP2SceneGraphWizard#110] [General]
Hi
I was reviewing the Certification criteria. 
https://developer.roku.com/docs/developer-program/certification/certification.md?lid=mfp9gkt4himg

the "freshest" version Last updated: April, 2024

Under 4. Channel operation i noticed: 

4.10 Channels must implement [bookmarking](https://developer.roku.com/docs/developer-program/media-playback/bookmarking.md) for all VOD content longer than 15 minutes. Channels must store bookmarks for a minimum of 30 days.
and 

and in 6. UI and Graphics
6.4 The Channel Store artwork and splash screen must clearly represent the name or identity of the channel using only broadcast-safe colors with proper sizing. The channel splash screen must support FHD (1920x1080p) and HD (1280x720p) resolutions. In addition, artwork must not be transparent. The splash screen's URI must be listed in the package manifest file.

So according to these, we should have a bookmarking option and submit the channel SDK with both FHD and HD graphics 

When you have some time, could you please look into these?
Thank you

**Comment by rrirower** [comment]
FYI...Bookmarking has been supported since the early versions of this software.  However, I was unaware of the 30-day limitation, so, I am updating the code to accommodate that criteria.

**Comment by rrirower** [comment]
Resolved in version 1.0.03.
